packages/gcf-utils/src/bin/genkey-util.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import fs from 'fs';
+import {Options} from 'probot';
+import {v1} from '@google-cloud/secret-manager';
+
+export async function gather(
+  keyfile: string,
+  id: number,
+  webhookSecret: string
+): Promise<Options> {
+  let keyContent = '';
+  try {
+    keyContent = fs.readFileSync(keyfile, 'utf8');
+  } catch (e) {
+    throw Error(`Error reading file: ${keyfile}`);
```

Comment:
This would just swallow the original exception.  I think you want to preserve the call stack and error message.  Generally in this case, I'd remove the try/catch and let it throw.  If you want to preserve the `Error reading file` bit, you can prepend to the origin `e.message`

Reply:
Done.